txscript/sign.go

```diff
@@ -288,7 +281,16 @@ sigLoop:
 			// bad input.
 			pkaddr := addr.(*btcutil.AddressPubKey)
 
-			pubKey := pkaddr.PubKey()
+			pubKeyOld := pkaddr.PubKey()
+
+			// A bit of modules induced weirdness: we need to
+			// conver types here since btcutil is still using the
```

Comment:
nit: cover to convert. But I guess this will be gone in 1777 anyways

Reply:
With the new version of https://github.com/btcsuite/btcd/pull/1785 -- should be the case that I no longer need to do this.